hashin.py

```diff
@@ -134,8 +137,9 @@ def run_single_package(
     )
     package = data['package']
 
+    maybe_restriction = '' if not restriction else ' ; {0}'.format(restriction)
```

Comment:
Shouldn't that be `'; {0}'.format(restriction)` not `' ; {0}'.format(restriction)`?